src/lib/blueprint.ts

```diff
@@ -1,15 +1,90 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  isRequired: boolean
+  description: string
+}
+
+interface Response {
+  description: string
+}
+
+type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
+
+interface Endpoint {
+  name: string
+  path: string
+  method: Method[]
+  semanticMethod?: string
```

Comment:
```suggestion
  semanticMethod: Method
```

Reply:
I'm realizing we need to encode the preferred method, e.g., we have [a lot of cases like this](https://github.com/seamapi/javascript-http/blob/bea6d987a58597692dc6b3640c39f953a979f819/test/seam/connect/seam-http-request.test.ts#L53-L73)

```
{
  path: '/devices/list',
  methods: ['GET', 'POST'],
  semanticMethod: 'GET',
  preferredMethod: 'POST'
}
```

Basically, we have to use POST because the API cannot support GET for all params yet.

Reply:
Does it make sense then to even mention other 'supported' methods? Maybe we should just stick to the `preferredMethod` for now?

Reply:
We need both `semanticMethod` and `preferredMethod` since, taking the js sdk as an example, `semanticMethod` determines some of the type naming (body vs params) but `preferredMethod` is used to determine the actual method used.

Having `methods` is useful IMO for us internally to track progress, e.g., we can compare the three properties to see which endpoints are missing methods, etc.